compiler/rustc_monomorphize/src/partitioning/default.rs

```diff
@@ -418,6 +419,7 @@ fn mono_item_visibility(
     let def_id = match instance.def {
         InstanceDef::Item(def) => def.did,
         InstanceDef::DropGlue(def_id, Some(_)) => def_id,
+        InstanceDef::AsRepr(def_id, _) => def_id,
```

Comment:
Can this one be hidden too? That will reduce the amount of symbols exported from dylibs and as such improve linker time when linking rust dylibs.